src/main/resources/fabric.mod.json

```diff
@@ -1,32 +1,13 @@
 {
   "schemaVersion": 1,
   "id": "cotton",
-  "version": "$version",
-
+  "version": "${version}",
   "name": "Cotton",
-  "icon": "assets/cotton/icon.png",
-  "description": "Common materials for inter-compatibility and ease of use.",
+  "icon": "icon.png",
+  "description": "Feature-rich set of libraries for the most important things you need",
   "licence": "MIT",
   "contact": {
-    "maven": "http://server.bbkr.space:8081/artifactory/libs-snapshot/io/github/cottonmc/cotton/",
-    "sources": "https://github.com/CottonMC/cotton",
+    "homepage": "https://curseforge.com/minecraft/mc-mods/cotton",
     "issues": "https://github.com/CottonMC/cotton/issues"
-  },
-
-  "environment": "*",
-  "entrypoints": {
-    "main": [
-      "io.github.cottonmc.cotton.Cotton"
-    ]
-  },
-  "mixins": [
-    "cotton.common.json"
-  ],
-  "requires": {
-    "fabricloader": ">=0.4.0",
-    "fabric": "*"
-  },
-  "custom": {
-    "modmenu:api": true
```

Comment:
We should still be marking this `modmenu:api`, shoudn't we? The fact that it's bundling the submods doesn't mean that it isn't an API.

Reply:
Yup, completely forgot about this one